### PR TITLE
Reformat code using black

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Reformat code using black

closes #62 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog

### CODE REVIEW

**Comment by tabergma** [comment]
Black is conflicting with the pep8 linter. Summarized all points here: https://github.com/RasaHQ/roadmap/issues/436.

File: rasa_core_sdk/endpoint.py [hunk 1]
```diff
@@ -89,52 +84,59 @@ def check_version_compatibility(core_version):
...
+            "You are using an old version of rasa_core which might "
+            "not be compatible with this version of rasa_core_sdk "
+            "({}).\n"
+            "To ensure compatibility use the same version "
+            "for both, modulo the last number, i.e. using version "
+            "A.B.x the numbers A and B should be identical for "
+            "both rasa_core and rasa_core_sdk."
+            "".format(rasa_core_sdk.__version__)
+        )
+    elif (
```
Comment (tmbo) [inline rasa_core_sdk/endpoint.py]: do you mind rewriting the condition to make it more readable?

File: rasa_core_sdk/forms.py [hunk 2]
```diff
@@ -173,8 +190,9 @@ def intent_is_desired(requested_slot_mapping, tracker):
         mapping_intents = requested_slot_mapping.get("intent", [])
         mapping_not_intents = requested_slot_mapping.get("not_intent", [])
         intent = tracker.latest_message.get("intent", {}).get("name")
-        return ((not mapping_intents and intent not in mapping_not_intents) or
-                intent in mapping_intents)
+        return (
```
Comment (tmbo) [inline rasa_core_sdk/forms.py]: this needs some rewriting as well, e.g. an intermediate var storing the first part of the `or`?

File: .travis.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ install:
 - pip install coveralls==1.3.0
 - pip list
 script:
-- py.test --pep8 -m pep8
+- if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install black; black --config rasa_core_sdk.toml --check .; fi
```
Comment (tmbo) [inline .travis.yml]: while we are at it let's move it to a separate job stage and put it under `jobs include`:
```
- stage: test
   name: Check code style
   python: 3.6
   script: pip install black; black --config rasa_core_sdk.toml --check . 
```
(no need for the if there, should only be run once)

File: rasa_core_sdk/cli/arguments.py [hunk 4]
```diff
@@ -6,27 +6,30 @@
...
+    )
     parser.add_argument(
-        '--cors',
-        nargs='*',
+        "--cors",
+        nargs="*",
         type=str,
-        help="enable CORS for the passed origin. "
-             "Use * to whitelist all origins")
+        help="enable CORS for the passed origin. " "Use * to whitelist all origins",
```
Comment (tmbo) [inline rasa_core_sdk/cli/arguments.py]:
```suggestion
        help="enable CORS for the passed origin. Use * to whitelist all origins",
```

File: rasa_core_sdk/__init__.py [hunk 5]
```diff
@@ -208,9 +218,9 @@ class ActionExecutionRejection(Exception):
 
     def __init__(self, action_name, message=None):
         self.action_name = action_name
-        self.message = (message or
-                        "Custom action '{}' rejected execution of"
-                        "".format(action_name))
+        self.message = message or "Custom action '{}' rejected execution of" "".format(
```
Comment (tmbo) [inline rasa_core_sdk/__init__.py]:
```suggestion
        self.message = message or "Custom action '{}' rejected execution of".format(
```

File: rasa_core_sdk/endpoint.py [hunk 6]
```diff
@@ -89,52 +84,60 @@ def check_version_compatibility(core_version):
...
+    edp_app = endpoint_app(cors_origins=cors, action_package_name=actions)
 
-    http_server = WSGIServer(('0.0.0.0', port), edp_app)
+    http_server = WSGIServer(("0.0.0.0", port), edp_app)
 
     http_server.start()
-    logger.info("Action endpoint is up and running. on {}"
-                "".format(http_server.address))
+    logger.info(
+        "Action endpoint is up and running. on {}" "".format(http_server.address)
```
Comment (tmbo) [inline rasa_core_sdk/endpoint.py]:
```suggestion
        "Action endpoint is up and running. on {}".format(http_server.address)
```

File: rasa_core_sdk/executor.py [hunk 7]
```diff
@@ -143,34 +149,33 @@ def register_package(self, package):
         try:
             self._import_submodules(package)
         except ImportError:
-            logger.exception("Failed to register package '{}'."
-                             "".format(package))
+            logger.exception("Failed to register package '{}'." "".format(package))
```
Comment (tmbo) [inline rasa_core_sdk/executor.py]:
```suggestion
            logger.exception("Failed to register package '{}'.".format(package))
```

File: rasa_core_sdk/executor.py [hunk 8]
```diff
@@ -143,34 +149,33 @@ def register_package(self, package):
...
     def run(self, action_call):
         action_name = action_call.get("next_action")
         if action_name:
             logger.debug("Received request to run '{}'".format(action_name))
             action = self.actions.get(action_name)
             if not action:
-                raise Exception("No registered Action found for name '{}'."
-                                "".format(action_name))
+                raise Exception(
+                    "No registered Action found for name '{}'." "".format(action_name)
```
Comment (tmbo) [inline rasa_core_sdk/executor.py]:
```suggestion
                    "No registered Action found for name '{}'.".format(action_name)
```

File: rasa_core_sdk/utils.py [hunk 9]
```diff
@@ -14,32 +14,34 @@ def all_subclasses(cls):
...
         const=logging.INFO,
         default=logging.INFO,
     )
     parser.add_argument(
-        '-vv', '--debug',
-        help="Print lots of debugging statements. "
-             "Sets logging level to DEBUG",
+        "-vv",
+        "--debug",
+        help="Print lots of debugging statements. " "Sets logging level to DEBUG",
```
Comment (tmbo) [inline rasa_core_sdk/utils.py]:
```suggestion
        help="Print lots of debugging statements. Sets logging level to DEBUG",
```

File: setup.py [hunk 10]
```diff
@@ -55,28 +49,27 @@
...
         "Source": "https://github.com/rasahq/rasa_core_sdk",
     },
 )
 
-print("\nWelcome to Rasa Core SDK!")
-print("If any questions please visit "
-      "documentation page https://docs.rasa.com/core")
-print("or join community chat on https://gitter.im/RasaHQ/rasa_core")
+print ("\nWelcome to Rasa Core SDK!")
+print ("If any questions please visit " "documentation page https://docs.rasa.com/core")
```
Comment (tmbo) [inline setup.py]:
```suggestion
print ("If any questions please visit documentation page https://docs.rasa.com/core")
```

**Pull request merged**